docs/paper.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'scores: A Python package for verifying accuracy using xarray and pandas'
```

Comment:
Consider if we have pandas in the title - not all metrics support pandas unless you convert them into an xarray object.

Reply:
Done